app/controllers/merge_requests_controller.rb

```diff
@@ -32,9 +32,8 @@ def update
     end
   end
 
-  def details
+  def show
     @merge_request = MergeRequest.find(params[:id])
```

Comment:
I think `before_action :find_resource` might do this anyway, so we might not need to define this method at all (?)